SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizer.h

```diff
@@ -57,6 +60,8 @@ namespace cms {
                                     edm::ConsumesCollector& iC);
     ~Phase2TrackerDigitizer() override;
 
+    void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
```

Comment:
`Phase2TrackerDigitizer` looks like a helper plugin, so this function has no effect unless something explicitly calls it.

In order to eventually have MixingModule to use [the helper plugin mechanism for `fillDescriptions()`](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#A_Plugin_Module_Using_Another_He), this would be better as
```suggestion
    static void fillPSetDescription(edm::ParameterSetDescription& descriptions);
```